string link_name
sensor_msgs/PointCloud2 bounding_box